src/pipecat/services/asyncai/tts.py

```diff
@@ -0,0 +1,513 @@
+                self._context_id = None
+            else:
+                logger.error(f"{self} error, unknown message type: {msg}")
+
+    async def _keepalive_task_handler(self):
+        """Send periodic keepalive messages to maintain WebSocket connection."""
+        KEEPALIVE_SLEEP = 10 if self.task_manager.task_watchdog_enabled else 3
+        while True:
+            self.reset_watchdog()
+            await asyncio.sleep(KEEPALIVE_SLEEP)
+            try:
+                if self._websocket and self._websocket.open:
+                    keepalive_message = {"transcript": " "}
+                    logger.trace("Sending keepalive message")
+                    await self._websocket.send(json.dumps(keepalive_message))
+            except websockets.ConnectionClosed as e:
+                logger.warning(f"{self} keepalive error: {e}")
+                break
+
+    @traced_tts
+    async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
+        """Generate speech from text using Async API websocket endpoint.
+
+        Args:
+            text: The text to synthesize into speech.
+
+        Yields:
+            Frame: Audio frames containing the synthesized speech.
+        """
+        logger.debug(f"{self}: Generating TTS [{text}]")
+
+        try:
+            if not self._websocket or self._websocket.closed:
+                await self._connect()
+
+            if not self._context_id:
+                await self.start_ttfb_metrics()
+                yield TTSStartedFrame()
+                self._context_id = self._global_context_id
+                await self.create_audio_context(self._context_id)
+
+            msg = self._build_msg(text=text)
+
+            try:
+                await self._get_websocket().send(msg)
+                await self.start_tts_usage_metrics(text)
+            except Exception as e:
+                logger.error(f"{self} error sending message: {e}")
+                yield TTSStoppedFrame()
+                await self._disconnect()
+                await self._connect()
+                return
+            yield None
+        except Exception as e:
+            logger.error(f"{self} exception: {e}")
+
+class AsyncAIHttpTTSService(TTSService):
+    """HTTP-based Async TTS service.
+
+    Provides text-to-speech using Async's HTTP streaming API for simpler,
+    non-WebSocket integration. Suitable for use cases where streaming WebSocket
+    connection is not required or desired.
+    """
+
+    class InputParams(BaseModel):
+        """Input parameters for Async API.
+
+        Parameters:
+            language: Language to use for synthesis.
+        """
+        language: Optional[Language] = Language.EN
+    
+    def __init__(
+        self,
+        *,
+        api_key: str,
+        voice_id: str,
+        aiohttp_session: aiohttp.ClientSession,
+        model: str = "asyncflow_v2.0",
+        url: str = "https://api.async.ai",
+        version: str = "v1",
+        sample_rate: int = 32000,
```

Comment:
Same pattern as above for sample rate
```suggestion
        sample_rate: Optional[int] = None,
```